src/fu-util-common.c

```diff
@@ -2445,6 +2445,26 @@ fu_util_security_event_to_string(FwupdSecurityAttr *attr)
 		      FWUPD_SECURITY_ATTR_RESULT_NOT_LOCKED,
 		      /* TRANSLATORS: HSI event title */
 		      _("UEFI memory protection is now unlocked")},
+		     /* ------------------------------------------*/
+		     {FWUPD_SECURITY_ATTR_ID_AMD_ENTRY_SIGN,
+		      FWUPD_SECURITY_ATTR_RESULT_UNKNOWN,
+		      FWUPD_SECURITY_ATTR_RESULT_NOT_LOCKED,
+		      /* TRANSLATORS: HSI event title */
+		      _("AMD microcode signature vulnerability detected")},
+		     {FWUPD_SECURITY_ATTR_ID_AMD_ENTRY_SIGN,
+		      FWUPD_SECURITY_ATTR_RESULT_UNKNOWN,
+		      FWUPD_SECURITY_ATTR_RESULT_LOCKED,
+		      NULL},
+		     {FWUPD_SECURITY_ATTR_ID_AMD_ENTRY_SIGN,
+		      FWUPD_SECURITY_ATTR_RESULT_NOT_LOCKED,
+		      FWUPD_SECURITY_ATTR_RESULT_LOCKED,
+		      /* TRANSLATORS: HSI event title */
+		      _("AMD microcode signature vulnerability fixed")},
```

Comment:
Considering there are CPUs not affected by this (Anything Zen 5 for example) I don't think `vulnerability fixed` is the right text.  How about instead:

`AMD microcode signature trusted`

Reply:
> Considering there are CPUs not affected by this

In that case the attribute won't be shown at all, no?